src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -79,6 +79,12 @@ public BuildScriptSnippet GenerateBashBuildScriptSnippet(BuildScriptGeneratorCon
                 configureYarnCache = true;
                 packageInstallerVersionCommand = NodeConstants.YarnVersionCommand;
             }
+            else if (IsHugoSite(ctx.SourceRepo))
```

Comment:
I am wondering if this would break existing apps in AppService or in general new apps' experience. For example, `config.json` is a very common file that any app could have. Here we would be considering it as a hugo app instead.

Reply:
I removed this scenario -- see above comment.